.stylelintrc

```diff
@@ -0,0 +1,135 @@
+{
+  "extends": "stylelint-config-standard",
+  "rules": {
+    "color-named": "never",
+    "declaration-no-important": true,
+    "font-family-name-quotes": "always-where-recommended",
+    "font-weight-notation": "numeric",
+    "function-url-quotes": "always",
+    "selector-max-specificity": "0,2,0",
+    "selector-no-id": true,
+    "selector-root-no-composition": true,
+    "string-quotes": "single",
+    "declaration-block-properties-order": [
```

Comment:
I don't usually object things when it comes to CSS. I think most of the rules here make sense.
The only one I don't agree with is `color-named`. I like CSS colour names because it's easier for a human being to understand what is the colour. But I'm more curious about understanding the rationale than challenging this rule.